cmd/data-crawler.go

```diff
@@ -755,6 +756,14 @@ func (i *crawlItem) applyActions(ctx context.Context, o ObjectLayer, meta action
 		return size
 	}
 
+	entry := audit.NewEntry(globalDeploymentID)
+	entry.Trigger = "internal-scanner"
+	entry.API.Name = "DeleteObject"
+	entry.API.Bucket = i.bucket
+	entry.API.Object = i.objectPath()
+	ctx = logger.SetAuditEntry(ctx, &entry)
+	logger.AuditLog(ctx, nil, nil, nil)
+
```

Comment:
This can be a function like `auditLogLifecycle()`

Reply:
> This can be a function like `auditLogLifecycle()`

Fixed